streaming/src/test/scala/org/apache/spark/streaming/util/WriteAheadLogSuite.scala

```diff
@@ -102,6 +105,114 @@ class WriteAheadLogSuite extends SparkFunSuite with BeforeAndAfter {
     }
   }
 
+  test("WriteAheadLog - read rotating logs") {
```

Comment:
Wait, do you even need it if different suites already show up different names. So probably we dont need "WriteAheadLog - " prefix at all. See if it makes sense.